crates/edr_op/src/lib.rs

```diff
@@ -18,18 +15,22 @@ pub mod hardfork;
 pub mod predeploys;
 /// Types for OP receipts.
 pub mod receipt;
-mod spec;
-pub use self::spec::OpChainSpec;
-
+/// OP RPC types
+pub mod rpc;
 /// Types for running Solidity tests.
 pub mod solidity_tests;
-
+mod spec;
+/// Utility types for testing.
+#[cfg(any(test, feature = "test-utils"))]
```

Comment:
What is the reason to define a `test-utils` feature instead of using `cfg(test)`